src/core/agent/services/multiSigService.ts

```diff
@@ -1093,6 +1096,219 @@ class MultiSigService extends AgentService {
 
     return { op, exnSaid: exn.ked.d };
   }
+
+  async offerPresentMultisigACDC(
+    multisigSignifyName: string,
+    notificationSaid: string,
+    acdcDetail: any,
+    issuerAidPrefix: string,
+    offerExnToJoin?: any
+  ) {
+    let exn: Serder;
+    let sigsMes: string[];
+    let dtime: string;
+
+    const exchangeMessage = await this.props.signifyClient
+      .exchanges()
+      .get(notificationSaid);
+
+    const applySaid = exchangeMessage.exn.d;
```

Comment:
We only need this if we aren't joining an existing offer from another multisig member. Can we move this and the `exchanges().get` to inside the if statement